File: 2114. Maximum Number of Words Found in Sentences/solution.js
/**
 * @param {string[]} sentences
 * @return {number}
 */
var mostWordsFound = function(sentences) {
    let rs = 0 ; 
    for( let item of sentences ){
        rs = Math.max( rs , item.split(" ").length ) ; 
    }
    return rs ; 
};

File: 2114. Maximum Number of Words Found in Sentences/solution.rs
impl Solution {
    pub fn most_words_found(sentences: Vec<String>) -> i32 {
        let mut rs : i32 = 0 ; 
        let n = sentences.len() ;
        for  i  in 0..n {
            let count  = sentences[i].split_whitespace().count() ; 
            rs = std::cmp::max( rs , count as i32 ) ; 
        }
        return rs ; 
    }
}